R/plotDendro.R

```diff
@@ -75,14 +78,23 @@ plotDendro <- function(decisionTree,
     segs <- as.data.frame(dendextend::get_nodes_xy(dendro))
     colnames(segs) <- c("xend", "yend")
 
-    # As label and which stat was used
-    # Labels will stack
+    # Add labels to nodes
     segs$label <- gsub(";", "\n", dendextend::get_nodes_attr(dendro, "label"))
+    segs$label <- sapply(segs$label, function(lab, maxFeaturePrint) {
```

Comment:
Use `vapply` not `sapply`. Use of `sapply` will result in warning in Bioconductor build system.